package.json

```diff
@@ -24,8 +24,11 @@
   },
   "dependencies": {
     "@bitwarden/cli": "^1.12.1",
+    "bitwarden-dmenu": "^1.5.9",
```

Comment:
I dont think we want to recursively include this!

Reply:
Lol sorry, I removed that in the latest commit :smile: